### ISSUE TITLE
PDF reader: Keyboard access to right side of toolbar and right-hand pane

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#2379]
https://forums.zotero.org/discussion/94838/beta-new-keyboard-navigation

We know about this, but just to have a ticket for it.

**Comment by mrtcode** [comment]
Do we want to be able to tab to the right pane, or there would a special keyboard shortcut for switching? And then how to go back to pdf-reader? Because, for example, in info pane you need to press tab many times to navigate through it.

**Comment by dstillman** [comment]
Maybe there could be a shortcut too, but for now, just Tab

And ideally you'd Shift-tab back out of the item pane, as in the library view, and also be able to tab out of it back to the beginning

**Comment by mrtcode** [comment]
So far I was unable to implement this, but continuing to work further.

I guess that wouldn't be enough if the order of navigation would be like this: Info/Note split button -> PDF reader -> Info/Note pane? `tabindex` doesn't work if iframes are involved, I might need to manually code that, if we want a different order.

Also, after which element it should jump out from PDF reader?

Navigating into info pane was never possible right? Navigating in other sections is also unpredictable, and I basically can't reach anything (in pre-beta Zotero as well).

I guess after we reactify everything, we'll implement a proper navigation from zero. But for now let's just define the minimal goals what we want to do here. Maybe just focusing to note editor is enough for now?
